.circleci/config.yml

```diff
@@ -25,6 +25,14 @@ jobs:
       - checkout      
       - run: npm install            
       - run: npm run ci
+      - run:
+          name: Define environment variable with lastest commit's message
+          command: |
+            echo 'export COMMIT_MESSAGE=$(git log -1 --pretty=format:"%s")' >> $BASH_ENV
+            source $BASH_ENV
+      - run:
+          name: Lint commit message
+          command: echo "$COMMIT_MESSAGE" | npx commitlint
```

Comment:
please run a check that this causes the cirlce-ci build to fail
if a bad commit message was used.